### ISSUE TITLE
"must attack if able" + "can't attack unless you pay" effects results in being unable to attack

### ISSUE THREAD

**Issue by sumsarz** [magefree/mage#7573]
I couldn't find another issue already opened for this.

Bug was observed with Thantis, the Warweaver and Koskun Falls on the battlefield in 4p EDH. I assume it works with other cards that provide the same effects.

Reportedly you were unable to attack the controller of Koskun Falls, even though you had enough mana to pay for any potential attackers.

**Comment by Dellg** [comment]
I think you pay mana only if you want to attack.

At least in Arena, you pay if you want to attack. I had a Haktos and my opponent had an Archon of Absolution on the battlefield. Haktos attack each combat if able, but the Archon says that I have to pay 1 for each attacking creatures. I had the mana to pay, but I chose not to pay.

**Comment by sumsarz** [comment]
Yes, that's how it should work. But it's not how it's working, thus the issue.

You can't _choose_ to attack someone with a Propaganda type effect with creatures that are forced to attack.
